spec/inject_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'inject'
+
+describe Array do
+  it "can add" do
+    expect([1,2,3].inject {|m,e| m + e} ).to eq 6
+    expect([1,2,3].my_inject {|m,e| m + e} ).to eq 6
+  end
+
+  it "can times" do
+     expect([1,2,3].inject {|m,e| m * e} ).to eq 6
```

Comment:
Use 2 (not 3) spaces for indentation.<br>Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.<br>Space inside parentheses detected.